### PR TITLE
Challenge

### PR DESCRIPTION
**Pull request by joshualloyd**
Finally stopped kicking the can and put together an initial version.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,28 +6,42 @@
...
-<section id="blog">
-</section>
-<footer>
-<h3>How to reach me</h3>
-<p>Email: address@domain.tld</p>
-<p>Github: <a href="#">Link</a></p>
-<p>Twitter: <a href="#">Link</p>
-<p>LinkedIn: <a href="#">Link</p>
-</footer>
+<div class="main-wrapper">
```
Comment (stevenwally) [inline index.html]: This should be indented as it is a child of the body.

File: index.html [hunk 2]
```diff
@@ -9,10 +9,10 @@
...
+		<section><h2 class="blue-underline">Bio</h2><p>I have always had an interest in technology and design. I graduated from MTSU in 2006 with a degree in graphic design. While I started doing work mostly in print, over the years I have picked up more and more knowledge and interest in web design and development. Finally in the spring of 2017, after being frustrated in the pace of my skill development. I made the decision to pivot to dive deeply into full-stack web development. I particulary am interested in using javascript everywhere since it has become such a widely implemented programing interface for a number of technologies.</p>
```
Comment (stevenwally) [inline index.html]: These section elements do not need to be indented as they are sibling elements to the header element.

File: index.html [hunk 3]
```diff
@@ -6,28 +6,42 @@
...
+					<img src="http://via.placeholder.com/200x150">
+					<h3>Project 2</h3>
+					<p>Here is upcoming project #2.</p>
+				</section>
+			</div>
+		</section>
+	</article>
+	<section id="blog"></section>
+	<footer>
+	<h2 class="blue-underline">How to reach me</h2>
```
Comment (stevenwally) [inline index.html]: Child elements of the footer element should also be indented.

**Review by stevenwally — APPROVED** [review]
Josh, everything looks great! I noticed a couple of indentation things that you should take a look at, but I am purposefully being nitpicky. Fix those when you have a chance, then you will be good to merge 👍

**Comment by joshualloyd** [comment]
Steve, thanks for the feedback. Fixed and merging.

**Pull request merged**